jsapp/js/components/languages/languageSelector.tsx

```diff
@@ -189,12 +189,12 @@ class LanguageSelector extends React.Component<LanguageSelectorProps, LanguageSe
   }
 
   clearSearchPhrase() {
-    this.setSearchPhrase('')
+    this.setSearchPhrase('', true)
   }
 
-  setSearchPhrase(searchPhrase: string) {
+  setSearchPhrase(searchPhrase: string, forceFetch?: boolean) {
     this.setState({ searchPhrase: searchPhrase })
-    if (searchPhrase.length >= MINIMUM_SEARCH_LENGTH) {
+    if (forceFetch || searchPhrase.length >= MINIMUM_SEARCH_LENGTH) {
```

Comment:
this looks like a bugfix unrelated to this PR, please extract as seperate PR.

Are other changes in this file and `EnvStore.ts` also bugfixes on their own, or neccessary only in the context of storybook testing?

Reply:
The rest of the changes in `languageStore.tsx` are needed for the tests